encrypted-media-respec.html

```diff
@@ -1661,30 +1663,28 @@
                         <dd>
                           <p>Run the following steps:</p>
                           <ol>
-                            <li><p>Clear all stored session data associated with this object, including the <a def-id="sessionId"></a> and license destruction record.</p>
-                              <p class="note">A subsequent call to <a def-id="load"></a> with the value of this object's <a def-id="sessionId"></a> would fail because there is no data stored for that session ID.</p>
-                            </li>
                             <li>
                               <p>
-                                Run the <a def-id="update-key-statuses-algorithm"></a> algorithm on the <var>session</var>, providing an empty sequence.
+                                Use <var>cdm</var> to clear all stored session data associated with this object, including the <a def-id="sessionId"></a>
+                                and license destruction record.
                               </p>
+                              <p class="note">A subsequent call to <a def-id="load"></a> with the value of this object's <a def-id="sessionId"></a> would fail because there is no data stored for that session ID.</p>
                             </li>
-                            <li><p>Run the <a def-id="session-close-algorithm"></a> algorithm on this object.</p></li>
+                            <li><p>Run the <a def-id="close-session-algorithm"></a> algorithm on this object.</p></li>
```

Comment:
In response to https://github.com/w3c/encrypted-media/pull/184/commits/251debc2d52c9a0202bfa8e7e26b16107def8d2f#r62881880:

What does it mean to close a session other than destroy the keys or removing session data (at least from an observable point of view)? I do not think it should be possible to get to this point without having gone through normative steps that closed the session but did not remove the session data. Thus, this seems redundant, especially since going back through the Close Session algorithm is confusing. (While not critical, it also adds another algorithm that is called both from the CDM and UA portion of algorithms.)